pkgs/development/compilers/swift/default.nix

```diff
@@ -2,11 +2,16 @@
 , pkgs
 , newScope
 , darwin
+, llvmPackages
 , llvmPackages_15
 , overrideCC
 }:
 
 let
+  libcxx = if lib.versionAtLeast llvmPackages.release_version "15"
+    then llvmPackages.libcxx
+    else llvmPackages_15.libcxx;
```

Comment:
At the risk of missing something obvious, I don't see where this local variable is used.

Reply:
That’s probably a leftover from a previous draft of the changes. I dropped it in the update I just pushed.